.github/workflows/bundle-size-comment.yml

```diff
@@ -0,0 +1,33 @@
+name: Bundle size | Comment on PR
+on:
+  workflow_run:
+    workflows: ['Bundle size']
+    types:
+      - completed
```

Comment:
don't you need to add the scoped permissions to the workflow?

Reply:
AFAIK - no. I will create another PR if it will not be enough